source/administration/changelog.md

```diff
@@ -4,6 +4,123 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ 
+#### Plugins
+ - Added support for markdown in plugin system console help text fields.
+ - Added support for plugins to override ephemeral posts.
+
+#### Localization
+ - Promoted Polish language to "official".
+
+#### Command Line Interface (CLI)
+ - Added a ``command modify`` CLI command to be able to modify slash commands.
+ - Added support for converting user accounts to bot accounts through the CLI.
+ - Implemented a new command ``MigrateConfig`` for migrating configuration to and from the database.
+ - For LDAP Groups, added the ability to switch a team or channel to be group-constrained via the CLI.
+ 
+#### Administration
+ - Added support for running two Mattermost instances on the same domain using subpaths.
+ - Added support for importing threads from Slack.
+
+### Bug Fixes
+ - Fixed an issue where releasing a mouse click while the cursor was outside of the Rename Channel modal would close the modal.
+ - Fixed an issue where a whitepage occured after uploading a plugin with an invalid ``settings_schema`` value.
+ - Fixed an issue where the announcement banner overlapped channel content.
+ - Fixed an issue where license expiration notice banner could not be dismissed prior to the license expiration date.
+ - Fixed an issue where the channel switcher autocomplete didn't function properly when autocompleting the name of a person who was the first person named in a group message channel.
+ - Fixed an issue where inline images in markdown preview didn't get expanded.
+ - Fixed an issue where replies to the parent post were not left-aligned.
+ - Fixed an issue where the timezone picker dropdown closed when trying to drag the scrollbar.
+ - Fixed an issue where the ``ExperimentalPrimaryTeam`` config.json setting no longer hid the "Leave Team" option.
+
+### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under ``"PluginSettings":`` in ``config.json``:
+   - Added ``"EnableHealthCheck": true``, to ensure all plugins are periodically monitored, and restarted or deactivated based on their health status.
+ - Under ``"NotificationLogSettings":`` in ``config.json``:
+   - Added ``"EnableConsole": true``, ``"ConsoleLevel": "DEBUG"``, ``"ConsoleJson": true``, ``"EnableFile": true``, ``"FileLevel": "INFO"``, ``"FileJson": true``, and ``"FileLocation": ""``, to implement a structured logger to keep track of push notifications.
+ - Under ``"ServiceSettings":`` in ``config.json``:
+   - Added ``"EnableBotAccountCreation": false`` and ``"DisableBotsWhenOwnerIsDeactivated": true``, to add the ability to enable bot accounts.
+   - Added ``"TrustedProxyIPHeader": []``, to explicitly define which IP headers are trusted.
+
+### Database Changes
+ - 
+
+### API Changes
```

Comment:
1) Propose breaking to "RESTful API Changes" and "Plugin API" changes (for plugins there is only one, the last one).

![image](https://user-images.githubusercontent.com/13119842/58820456-aa7dbd80-8600-11e9-82de-faae02c3d453.png)

This helps make it more clear which APIs the changes refer to.

2) Propose formatting the API changes consistently, similar to the screenshot above. Let me know if I can help.